lib/rubocop/cop/rails/presence.rb

```diff
@@ -76,6 +76,8 @@ def on_if(node)
             receiver, other = redundant_negative_receiver_and_other(node)
           end
           return unless receiver
+          return if other.multiline? || other.if_type? || other.rescue_type?
```

Comment:
Do not use `other.multiline?`. Probably the cop breaks on multiple statement on one line. For example:

```ruby
if a.present?
  a
else
  something; something; something
end
```


We can check `begin` type for other node instead.

```ruby
          {
            (if
              (send $_recv :present?)
              _recv
              $!begin
            )
...
```

Because if `if` body has multiple statements, the body is a `begin` node.

```bash
$ ruby-parse -e if cond then foo end
(if
  (send nil :cond)
  (send nil :foo) nil)

$ ruby-parse -e if cond then foo; foo end
(if
  (send nil :cond)
  (begin
    (send nil :foo)
    (send nil :foo)) nil)
```